packages/nimble-components/.prettierignore

```diff
@@ -4,16 +4,21 @@
 !*.ts
 !*.js
 !*.md
+!*.mdx
 !*.json
 !src/
+!docs/
 !/.storybook/
 
 /src/**/*.*
 !src/**/*.ts
 !src/**/*.js
 !src/**/*.md
+!src/**/*.mdx
 !src/**/*.json
 
+!docs/**/*.mdx
```

Comment:
To resolve the lint error you need to follow the pattern like src above (first opt-out all files and then selectively opt-in desired files)

```suggestion
/docs/**/*.*
!docs/**/*.mdx
```